src/main/java/org/datadog/jmxfetch/App.java

```diff
@@ -136,6 +136,11 @@ public static void main(String[] args) {
 
         App app = new App(config);
 
+        // Get config from the ipc endpoint for "list_*" actions
+        if (!config.getAction().equals(AppConfig.ACTION_COLLECT)) {
```

Comment:
Wondering if this might break stuff on agent5? Can you verify? Maybe we should add a flag to enable this behavior?

Reply:
It should not since the A5 does not give the IPC config parameters . https://github.com/DataDog/jmxfetch/blob/1e163cf616758b2df52ac311d0183cf39c2c8c9c/src/main/java/org/datadog/jmxfetch/App.java#L538
^ `getJSONConfigs` is a no-op in A5